resource/graph_node.go

```diff
@@ -570,6 +571,36 @@ func (w *GraphNode) resourceStatus() Status {
 	}
 }
 
+type graphNodeStats struct {
+	State    int
+	ResStats any
+}
+
+func (w *GraphNode) Stats() any {
+	ret := graphNodeStats{}
+
+	res, err := w.Resource()
+	switch err {
+	case nil:
+		ret.State = 0
+	case errNotInitalized:
+		ret.State = 1
+	case errPendingRemoval:
+		ret.State = 2
+	default:
+		// `w.lastErr != nil`
+		ret.State = 3
+	}
+
+	if statser, isStatser := res.(ftdc.Statser); isStatser && err == nil {
+		ret.ResStats = statser.Stats()
+	} else {
+		ret.ResStats = struct{}{}
```

Comment:
I can experiment with removing this again. This was part of the whole "oh crap I didn't support nested `any`".

Reply:
I took this out, I think `nil` is better.